src/components/dashboard-selector/dashboard-selector.directive.coffee

```diff
@@ -17,8 +17,8 @@ angular
       # ============================================
 
       $scope.organizationsNames = ->
-        _.pluck $scope.currentDhb.data_sources, 'label'
-        .join ", "
+        sortedOrgNames = _.sortBy($scope.currentDhb.data_sources, (org) -> org.label.toLowerCase())
+         _.pluck(sortedOrgNames, 'label').join(", ")
```

Comment:
Hey Ill fix this up on the release branch, but just note the indentation is out by one space - enough to break coffeescript compile with "unexpected indentation".

Reply:
@xaun Sorry about that!